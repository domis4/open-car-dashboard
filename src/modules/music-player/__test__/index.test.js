import React from 'react'
import { shallow } from 'enzyme'
import MusicPlayer from '../index'

describe('music-player module index.js', () => {
  it('expects component to match snapshot', () => {
    const wrapper = shallow(<MusicPlayer />).dive()

    expect(wrapper).toMatchSnapshot()
  })
})
