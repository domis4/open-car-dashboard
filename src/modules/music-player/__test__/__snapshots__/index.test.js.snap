// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`music-player module index.js expects component to match snapshot 1`] = `
<View
  classes={
    Object {
      "bottom": "withState-View--bottom-02",
      "root": "withState-View--root-01",
    }
  }
  setTabValue={[Function]}
  tabValue="lists"
/>
`;
